lib/datasource/rubygems/releases.ts

```diff
@@ -3,15 +3,21 @@ import { getDependency } from './get';
 import { getRubygemsOrgDependency } from './get-rubygems-org';
 import { PkgReleaseConfig, ReleaseResult } from '../common';
 
+const isDefaultUrl = (url: string): boolean => url.endsWith('rubygems.org');
+
 export async function getPkgReleases({
   lookupName,
   registryUrls,
 }: PkgReleaseConfig): Promise<ReleaseResult | null> {
-  const registries = is.nonEmptyArray(registryUrls) ? registryUrls : [];
+  const defaultRegistry = 'https://rubygems.org';
+  const registries = is.nonEmptyArray(registryUrls)
+    ? registryUrls.filter(url => !isDefaultUrl(url))
+    : [];
```

Comment:
I expected that we’d only add the default registry of the registryUrls is empty. Otherwise it means it’s impossible to “block” rubygems lookups, right?

Reply:
Right, my concern here is to avoid accidental blockings, in cases when Gemfile contains only third-party urls. Now I doubt that's right assumption.

Reply:
In most package managers there is a way for enterprises to block using the default public registry

Reply:
👍 I've modified it, also it looks not so radical now